SUMMARY.md

```diff
@@ -0,0 +1,116 @@
+# Summary
```

Comment:
Was this file generated automatically, or should we note in our process that we need to update it every time?

Reply:
Nope, we'd need to manually update SUMMARY.md whenever new .md files are added, we'd need to add that to the contribution guidelines